source/jormungandr/jormungandr/scenarios/new_default.py

```diff
@@ -546,20 +546,26 @@ def _tag_by_mode(responses):
         for j in r.journeys:
             _tag_journey_by_mode(j)
 
-def _switch_back_to_ridesharing(response):
+
+def _is_fake_car(i, section, sections, dep_mode, arr_mode):
```

Comment:
please rename that `i` param 🙏  (if you keep the code that way)
also it's redundant to have `index and sections` on one side, and `section` on the other side